.gitignore

```diff
@@ -1,3 +1,4 @@
 /composer.lock
 /plugins
 /vendor
+/.idea
```

Comment:
This should go to your environment's global ignore file. Please don't modify this repo's .gitignore.